src/main/java/com/lithium/dbi/rdbi/recipes/scheduler/ExclusiveJobSchedulerDAO.java

```diff
@@ -71,7 +71,7 @@ int scheduleJob(
     @Query(
         "local deletedFromReadyQueue = redis.call('ZREM', $readyQueue$, $job$)\n" +
         "local deletedFromRunningQueue = redis.call('ZREM', $runningQueue$, $job$)\n" +
-        "if not deletedFromReadyQueue and not deletedFromRunningQueue then\n" +
+        "if deletedFromReadyQueue == 0 and deletedFromRunningQueue == 0 then\n" +
```

Comment:
It was probably a bug, but please install 2.8.x (latest) locally as that is what we now use in most of our testing & prod.